### ISSUE TITLE
The "archive" MIME type definition excludes some "zip" MIME types.

### ISSUE THREAD

**Issue by nuno-andre** [whatwg/mimesniff#140]
In [4.6. MIME type groups](https://mimesniff.spec.whatwg.org/#mime-type-groups), _ZIP_ is defined as having an `application/zip` essence or a `zip` suffix.

In the following paragraph, _archive_ is described as having `application/zip`, `application/x-rar-compressed` or `application/x-gzip` as an essence, but no mention is made of the `zip` suffix. 

Maybe a definition similar to that of _scriptable_ would be clearer:

> An archive MIME type is a ZIP-based MIME type or any MIME type whose essence is one of the following:
>
> - application/x-rar-compressed
> - application/x-gzip 

https://github.com/whatwg/mimesniff/blob/609a3a3c935fbb805b46cf3d90768d695a1dcff2/review-drafts/2020-01.bs#L424-L432

**Comment by GPHemsley** [comment]
You are correct that the **archive MIME type** definition does not include the full **ZIP-based MIME type** definition. It is possible that that was an oversight, but it's also possible that it wasn't.

**ZIP-based MIME type** is not actually used anywhere in the spec, and **archive MIME type** is only used as part of the **archive type pattern matching algorithm**. This algorithm, as written, is never going to detect a MIME type with an essence having a `+zip` suffix, so it is potentially unnecessary to include such a MIME type in the definition of **archive MIME type**.

(I'll leave it to someone else to make a final determination.)

**Comment by nuno-andre** [comment]
So _archive type_ is somewhat of a misnomer, because `.zip` files are indeed archives, but `.gz` isn't. Maybe _compressed type_ would be a more appropriate name?

Cheers!
